Data/ByteString/Builder/ASCII.hs

```diff
@@ -35,9 +35,6 @@ module Data.ByteString.Builder.ASCII
     , word64Dec
     , wordDec
 
-    , floatDec
-    , doubleDec
```

Comment:
This is a public module, we'd rather keep its API intact. Please import `Data.ByteString.Builder.RealFloat` and re-export `floatDec` / `doubleDec` from it.